src/AssetsPage/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import AssetsTable from './AssetsTable';
+import { AssetsTable } from './AssetsTable';
```

Comment:
Is this change as well as the AssetsTable/index change to export the class going to be necessary for all future component classes?

Reply:
In order to write tests for the actual React component, you need to be able to import the non-connected component, because, otherwise, jest complains about not having a Redux store. So, instead of importing the WrappedAssetsTable, I need to import the AssetsTable; therefore, I have to do a named export of AssetsTable. Does that make sense? If we want to test our components independent of their Redux stores, then, yes, we'll have to do this for each component, as we write unit tests for them. I believe you can test Redux separately anyway, but I'll defer to Ari on the best way to position ourselves relative to testing going forward.

Reply:
That does make sense, thanks! Wanted to sanity check understanding on what steps would be needed for future testing.